components/about.tsx

```diff
@@ -1,81 +1,101 @@
-              fill="url(#illustration-02)"
-            />
-          </svg>
-        </div>
+import React from "react";
+import { Card, CardContent, CardDescription } from "@/components/ui/card";
 
-        {/* Hero content */}
-        <div className="relative pb-10 pt-32 md:pb-16 md:pt-40">
-          {/* Section header */}
-          <div className="mx-auto max-w-3xl pb-12 text-center md:pb-16">
-            <h1 className="h1 mb-4" data-aos="fade-up">
-              Run by nang and FryingPan
-            </h1>
-            <p
-              className="mb-8 text-xl text-gray-600"
-              data-aos="fade-up"
-              data-aos-delay="200"
-            >
-              Building in Public
+const AboutComponent: React.FC = () => {
+  return (
+    <section
+      className="to-white bg-gradient-to-b from-gray-50 px-4 py-12 pt-20"
+      data-aos="fade-up"
+      data-aos-delay="200"
+    >
+      <div className="mx-auto max-w-6xl">
+        <div className="flex flex-col lg:flex-row">
+          {/* Left column */}
+          <div className="lg:w-1/2 lg:pr-8">
+            <h2 className="mb-4 text-3xl font-bold">
+              Fully <span className="text-primary-700">transparent</span>,
+              PearAI is built in public
+            </h2>
+            <p className="mb-4 text-gray-600">
+              PearAI&apos;s goal is to reduce the time it takes for an
+              individual to go from idea to creation. Coding itself is a crucial
+              tool in product development, and we believe that with the
+              advancement of AI, it may drastically change over the coming
+              years. We aim to build the environment that will encompass these
+              changes, both in the short-term and long-term.
             </p>
-            <div className="mx-auto max-w-xs sm:flex sm:max-w-none sm:justify-center sm:gap-4">
-              <div data-aos="fade-up" data-aos-delay="400">
-                <a
-                  className="btn mb-4 w-full bg-primary-500 text-white-main hover:bg-primary-600 sm:mb-0 sm:w-auto"
-                  href="https://www.youtube.com/nang88"
-                  target="_blank"
-                >
-                  Watch Nang
-                </a>
-              </div>
-              <div data-aos="fade-up" data-aos-delay="400">
-                <a
-                  className="btn mb-4 w-full bg-secondary-main text-white-main hover:bg-gray-800 sm:mb-0 sm:w-auto"
-                  href="https://www.youtube.com/@PansDay"
-                  target="_blank"
-                >
-                  Watch Pan
-                </a>
-              </div>
+            <h3 className="mb-2 text-xl font-semibold">
+              Current features include:
+            </h3>
+            <ul className="mb-8 list-inside list-disc text-gray-600">
```

Comment:
```suggestion
            <ul className="mb-8 list-inside list-disc text-secondary-400">
```
We should use secondary colors instead of gray to have consistent styling